Makefile

```diff
@@ -22,11 +22,16 @@ build:
 	  $(MAKE) $(BINDIR)/$$i; \
 	done
 
-build/mackerel-plugin:
+build/mackerel-plugin: $(patsubst %,depends_on,$(GOOS)$(GOARCH))
 	mkdir -p build
+	rm -f build/mackerel-plugin
```

Comment:
Do you need this code?  I think we need either `$(patsubst %,depends_on,$(GOOS)$(GOARCH))` or `rm -f build/mackerel-plugin`

Reply:
No, I will drop this line.